envSample

```diff
@@ -0,0 +1,6 @@
+DB_USERNAME_DEVELOPMENT 
+DB_PASSWORD_DEVELOPMENT
+DB_NAME_DEVELOPMENT
+DB_HOSTNAME_DEVELOPMENT
+NODE_ENV
+PORT
```

Comment:
Good Job so far.
A `.env.example` file has already been created, so I think you should just make use of that one.

Reply:
done